ansible/www-standalone/resources/config/nodejs.org

```diff
@@ -32,19 +32,19 @@ server {
     default_type text/plain;
     index index.html;
 
-    # We set a default cache of 1 hour and a max age of 4 hours
+    # We set a default cache for browsers of 1 hour, and ask Cloudflare to cache for 4 hours
     add_header Cache-Control "public, max-age=3600, s-maxage=14400";
 
     error_page 404 /404.html;
 
-    # For the unencrypted Node.js server, if a request is not done within the /dist folder
-    # We basically rewrite/redirect the request to the https:// schema/protocol
+    # For the unencrypted Node.js server, we redirect any request to the HTTPS site
+    # Unless it is within /dist or is for a JSON file
```

Comment:
Would it make sense to comment why we don't redirect for JSON files?

Reply:
I don't know. That might be a historical reason. cc @richardlau @targos

Reply:
I have no idea.